### PR TITLE
Docker compose file creation

### PR DESCRIPTION
**Pull request by mpecarina**
## What It Does

Enable local development using Windows-friendly file paths.

## How To Test
docker-compose up
docker-compose down

docker-compose --build --force-recreate

docker-compose run node-sso-formatter yarn test:e2e
docker-compose run node-sso-formatter yarn build && yarn mock && yarn dev 
docker-compose down

## Notes/Caveats

@svetkomir  also updated a fork of Binci to be friendlier to Windows but there have been some issues with antivirus.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      && yarn build
+      && yarn mock
+      && yarn dev"
+  vault:
+    image: registry.banno-internal.com/node-vault-dev:latest
+    ports:
+      - 8200:8200
+    environment:
+      - VAULT_DEV_LISTEN_ADDRESS=0.0.0.0:8200
+      # - VAULT_DEV_ROOT_TOKEN_ID=root
```
Comment (brandonros) [inline docker-compose.yml]: nit: wanna remove this?

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,58 @@
+version: "3.7"
+services:
+  node-sso-formatter:
+    image: registry.banno-internal.com/node-dev:8
+    ports:
+      - 8080:8080
+      - 9090:9090
+    environment:
+      # General
```
Comment (brandonros) [inline docker-compose.yml]: can you think of any way around having to manage this in multiple places?

**Comment by brandonros** [comment]
> Will there be a future PR that updates the docs and rips out Binci?

I think the idea is to have both.

**Comment by mpecarina** [comment]
yea my thought was not to rip anything out in case the Windows users can only use Compose... we can always revisit this

**Comment by brandonros** [comment]
Do we need this if Svet was able to edit Binci in his fork?

**Comment by mpecarina** [comment]
It would be simpler not to have it, but until I know how Svet writes his windows HOW-TO, I would like it available as an option to him. He has an open ticket with corporate that should determine whether he develops in a Linux VM or Windows native.

File: docker-compose.yml (hunk 2 above)
Comment (mpecarina) [reply docker-compose.yml]: docker-compose can load our (dev) environment variables from file, it should be possible to maintain them in one place for both compose and binci

**Pull request merged**